contrib/gcppubsub/config/README.md

```diff
@@ -40,16 +41,18 @@ They do not offer:
 
    1. Determine the Service Account to use, or create a new one.
 
-    If using existing service account, just set an env variable.
-    ```shell
-	export PUBSUB_SERVICE_ACCOUNT='YOUR_SERVICE_ACCOUNT_NAME'
-    ```
+   If using existing service account, just set an env variable.
+
+   ```shell
+   '
```

Comment:
These seem wrong, it looks like dropped the `export` statement.

Reply:
it does...